features/provider/integration/edit.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Edit Integration
+  In order to integrate with 3scale
+  As a provider
+  I want to be able to edit the integration
+
+  Background:
+    Given a provider is logged in
+     And all the rolling updates features are off
+
+  Scenario: Edit a tested integration has a link to the analytics usage
+    Given the service has been successfully tested
+    When I go to the service integration page
+     And I follow "the analytics section"
+    Then I should see "Usage"
```

Comment:
Hmmm probably there would be more than one "Usage" in the page
Can we have something more relevant to check?

Reply:
Then I think I should check that it is in the title, because the content itself is unnecessarily complicated for this (it is the stats graphics)

Reply:
Hmmm can you just click on the link and see that it leads you to the right page?

Reply:
I clicked on the link in `I follow "the analytics section"`
the `see that it leads you to the right page` is what is difficult here and what I am trying to do 🤔 Btw, this is outdated and it is not the current code... look at the current one and let me know what you propose otherwise.